app/Http/Controllers/Api/RequestApiController.php

```diff
@@ -50,7 +49,7 @@ public function storeRequest(Request $request): Response
     {
         $this->validate($request, [
             'url' => 'required|url',
-            'method' => 'required|in:'.RequestHeaderParser::getAllowedMethodsImploded(),
+            'method' => 'required|in:'.implode(',', \App\Request::getAllowedMethods()),
```

Comment:
We discussed this. It's not ideal putting this in the model class, but it will do for now.